src/lib/eliom_common.server.ml

```diff
@@ -89,14 +89,14 @@ type cookie_exp =
   | CEBrowser   (** expires at browser close *)
   | CESome of float (** expiration date *)
 
-(* 2013-03-01 From now on, cookie expire 10 years after opening the session.
+(* By default cookies expire 1 year after opening the session.
    Before, it was when the browser was closed but we think it has no sense,
    and many people do not understand why their session is closed, even if
    the session duration on server side is long.
    If you want this, you now have to set this manually.
 *)
 let default_client_cookie_exp () =
-  CESome (Unix.time () +. 315532800.)
+  CESome (Unix.gettimeofday () +. 31553280.)
```

Comment:
I don't think this is a good idea to do that now.
We should first have a mechanism to refresh cookies.
Otherwise, we will start to have many people getting unsuspectingly logged-out one year from now.

Also, essential cookies are ok with the law for most use cases in most countries.

Reply:
Oh, I had thought that cookies were already being refreshed!

Reply:
The default server-side timeout is one day (see server/eliommod_timeouts.ml).
The client-side cookies become useless once they expire server-side.